.github/workflows/code-coverage.yml

```diff
@@ -31,13 +31,15 @@ jobs:
         dotnet-version: |
           6.0.x
           7.0.x
+          8.0.x
         source-url: https://nuget.pkg.github.com/open-feature/index.json
 
     - name: Run Test
       run: dotnet test --verbosity normal /p:CollectCoverage=true /p:CoverletOutputFormat=opencover
 
-    - uses: codecov/codecov-action@v3.1.6
+    - uses: codecov/codecov-action@v4.3.0
       with:
         name: Code Coverage for ${{ matrix.os }}
         fail_ci_if_error: true
```

Comment:
I wonder if we should make this false? Do we want CI to fail if there's an issue with CodeCov?

Reply:
I argue yes. Usually, we aim for higher code coverage and if it fails, we need to justify to the SDK admins why